src/main/scala/org/broadinstitute/dsde/firecloud/dataaccess/HttpGoogleServicesDAO.scala

```diff
@@ -69,9 +69,10 @@ object HttpGoogleServicesDAO {
       case null => None
       case x => Some(x)
     }
-    val idToken = gcsTokenResponse.getIdToken match {
-      case null => None
-      case x => Some(x)
+    val List(idToken:Option[String], subjectId:Option[String]) = gcsTokenResponse.getIdToken match {
+      case null => List(None, None)
+      case x =>
+        List( Some(x), Some(gcsTokenResponse.parseIdToken().getPayload.getSubject) )
     }
```

Comment:
I'm pretty sure the `List`s here are extraneous - I haven't seen this exact form of multiple assignment. This could be 
```
    val (idToken:Option[String], subjectId:Option[String]) = gcsTokenResponse.getIdToken match {
      case null => (None, None)
      case x =>
        (Some(x), Some(gcsTokenResponse.parseIdToken().getPayload.getSubject))
    }
```